### PR TITLE
Hotline number

### PR DESCRIPTION
**Pull request by bhavyaab**
## Description
- hotlinePhoneNumber field and verify hotlinePhoneNumber field added to the form
- update field property on receiving new resource.hotlinePhoneNumber
- It saves data in the database on clicking save
NOTE: resource.hotlinePhoneNumber is currently not changing on each new resource change (backend error)
## Test Plan
- run yarn webpack
- go to http://localhost:4000/verification/pregnancy-resource-center
- click 'Verify Next Resource' button
- hotlinePhoneNumber field is on the form immediately below the Phone Number field
- all design details match with the Phone Number field
- placeholder is 'Hotline Phone Number'
- update the filed and click save 
- reload the browser 
- click 'Verify Next Resource' button until find the same resource you just have updated
- see hotlinePhoneNumber filed it is populated 
NOTE: now on clicking Verify Next Resource button new resource will populate but hotlinePhoneNumber will not update because of resource.hotlinePhoneNumber is not updating in getResourceToVerify component. (Error)

### CODE REVIEW

**Comment by karenmrose** [comment]
@bhavyaab Excellent detail in the description and test plan! 👏 👏 👏 

The reason why it's not saving is because the phone number needs to be formatted correctly on the front-end. See this code. Let me know if you have any questions!

**Comment by bhavyaab** [comment]
About not clearing the hotlineNumber input field, I have mentioned in the PR details NOTE, that it will not clear out because when we go to next resource it is still getting resource.hotlineNumber = previous houtlineNumber instead if  getting resource.hotlineNumber = undefined or null

File: server/test/fqhc-tests.js [hunk 1]
```diff
@@ -248,6 +248,7 @@ describe('FQHCs', () => {
 				fqhcId: oldFQHC._id,
 			})
 			const fields = _.map(histories, 'field')
+			log.info(fields)
```
Comment (karenmrose) [inline server/test/fqhc-tests.js]: @katelynsills Do we need this checked into master or was this just for testing?

Comment (katelynsills) [reply server/test/fqhc-tests.js]: Whoops, we can remove it

**Comment by karenmrose** [comment]
@bhavyaab Is there something you wanted to talk about or should this be "needs review" now?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -3,15 +3,16 @@
 1.  Request access to this repo
 
 1.  Pull the latest master
-    - On the `master` branch: `git pull`
+
```
Comment (karenmrose) [inline CONTRIBUTING.md]: @bhavyaab Can you remove this file from this PR?

File: client/js/hah-app/reducers.js [hunk 3]
```diff
@@ -27,6 +27,7 @@ const authenticationReducer = (state = {}, action) => {
 const resourceReducer = (state = {}, action) => {
 	switch (action.type) {
 		case GET_RESOURCE_TO_VERIFY:
+			state.hotlinePhoneNumber = action.resource.hotlinePhoneNumber
```
Comment (karenmrose) [inline client/js/hah-app/reducers.js]: @bhavyaab We can't update state directly in redux 
https://redux.js.org/recipes/structuringreducers/immutableupdatepatterns

Don't worry about this not updating correctly. I'm working on a separate PR to clear the entire form on save and when you load a new resource. We keep running into this problem and it's a really major issue so I want to make sure it's solved for every single field on the form.

**Pull request merged**